tests/func/api/test_show.py

```diff
@@ -0,0 +1,404 @@
+import json
+import os
+from textwrap import dedent
+from typing import Dict, List
+
+import pytest
+
+from dvc import api
+from dvc.exceptions import DvcException
+
+FIXTURE_METRICS: Dict[str, List[Dict[str, Dict[str, float]]]] = {
```

Comment:
minor: Like the idea of the constant on top but the current definition makes it a little verbose when used in the tests, like 

```
FIXTURE_METRICS["test_metrics"][1][
            "avg_prec"
        ]
```

Maybe split into `TRAIN_METRICS` and `TEST_METRICS`?

Reply:
Thanks for the suggestion. I modified the code to use your suggested variable names.